### ISSUE TITLE
Please consider colorblind accesibility in the mineral overlay

### ISSUE THREAD

**Issue by Bystander333** [MaFi-Games/Captain-of-Industry-issues#787]
Hi, I love the game but really struggle with the mineral overlay. I'm red/green colorblind (red weak) - about 8% of men have some colorblindness. I generally have to send out a miner to figure out what the resource is rather than guess from the overlay.
Started playing "you shall not pass" and don't have time to do that. I've attached a Daltonize simulation of what the mineral overlay looks like to me - I assume it worked but to be clear this image and the game look identical as far as I'm concerned.
![COIC](https://user-images.githubusercontent.com/3098892/189771312-5153eb91-dd6f-4e8b-b040-f344de98b7b2.png)
This is using daltonlens https://github.com/DaltonLens/DaltonLens-Python with Mikado 2009.

Basically if you compare the key on the left with the bars. The coal could be oil, the back left (iron?) matches coal, the lighter stuff at the back right is one of sand, limestone or copper.

Fixing color across all variations of color blindness is tough - but adding something else e.g. hover over text saying "iron" or more distinctive patterns should be an easy fix.
